css/main.css

```diff
@@ -51,6 +52,42 @@ header {
   color: white;
 }
 
+#mobile-nav {
+  background-color: #3c3a39;
+  display: none;
+  height: 100vh;
+  position: fixed;
+  inset: 0 0 0 0;
+  z-index: 10;
+}
+
+#mobile-nav > ul {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  align-items: start;
+  margin: 0 1rem;
+}
+
+.menu-close {
+  color: #67798e;
+  position: absolute;
+  right: 0;
+  margin: 0 1rem;
+}
```

Comment:
**Required Changes** :warning: 

- [ ] Thank you for implementing the changes required but your design is still different from the Figma design. Kindly follow the Figma design files to make it look perfect and get the most professional results. Hint: You can get the specific properties of any element in Figma by clicking on it.
- [ ] You close the button on the same line, Kindly try to fix the position.

I attach an image with the part that you should look at.

![sahar re](https://user-images.githubusercontent.com/53347966/169516438-7d54447e-a938-4f34-8892-8a38010c7706.png)